portingdb/load_data.py

```diff
@@ -297,3 +297,13 @@ def load_from_directories(data, directories):
             package['ftbfs_age'] = max(CURRENT_FEDORA - releasever, 0)
         else:
             package['ftbfs_age'] = 0
+
+    # Consolidate non-Python requirers
+    for name, package in packages.items():
+        nonpy_requirers = {}
+        package.setdefault('needs_py2_command', False)
+        for rpm in package['rpms'].values():
+            for kind, names in rpm.get('non_python_requirers', {}).items():
+               if names:
+                    nonpy_requirers.setdefault(kind, set()).update(names)
+        package['non_python_requirers'] = nonpy_requirers
```

Comment:
I've read this code 5 times and I have no idea what does it actually do. Sorry. I must miss some context.

Reply:
Oh, so we already track this in

https://github.com/fedora-python/portingdb/blob/4bce049022122493e4ce813bf20cf2224a85e0ad/dnf-plugins/py3query.py#L353-L356

Reply:
Yes, but that's per RPM.
We probably don't need it per-RPM, but I'd like to keep old `fedora.json` formats loadable, so it's probably better to postprocess in load_data.